Tests/SonarScanner.MSBuild.TFS.Test/ProgramTests.cs

```diff
@@ -107,12 +107,5 @@ public void Execute_ShouldExecute_SummaryReportBuilder_ShouldSucceed()
         var result = Program.Execute(["SummaryReportBuilder", Path.Combine(tempDir, "temp.xml"), Path.Combine(tempDir, "sonar-project.properties"), "true"], logger);
 
         result.Should().Be(0);
-        logger.Errors.Should().HaveCount(1);
```

Comment:
This didn't made sense. The test is "ShouldSucceed", and this is emitted only because the scaffolding was missing the project info xml file.

If there were no files to found, there would already be that fat warning from the normal scanner run. It's simply not needed in this TFS thing, as the "nothing found" will be visible in the summary anyway.